xml/System/Array.xml

```diff
@@ -3517,6 +3517,25 @@
       <Docs>
         <summary>Searches for an element that matches the conditions defined by a specified predicate, and returns the zero-based index of the last occurrence within an <see cref="T:System.Array" /> or a portion of it.</summary>
       </Docs>
+      <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Examples
+ The following code example demonstrates all three overloads of the <xref:System.Array.FindLastIndex%2A> generic method. An array of strings is created, containing 8 dinosaur names, two of which (at positions 1 and 5) end with "saurus". The code example also defines a search predicate method named `EndsWithSaurus`, which accepts a string parameter and returns a Boolean value indicating whether the input string ends in "saurus".
+
+ The <xref:System.Array.FindLastIndex%60%601(%60%600[],System.Predicate{%60%600})> method overload traverses the array backward from the end, passing each element in turn to the `EndsWithSaurus` method. The search stops when the `EndsWithSaurus` method returns `true` for the element at position 5.
+
+> [!NOTE]
+>  In C# and Visual Basic, it is not necessary to create the `Predicate<string>` delegate (`Predicate(Of String)` in Visual Basic) explicitly. These languages infer the correct delegate from context and create it automatically.
+
+ The <xref:System.Array.FindLastIndex%60%601(%60%600[],System.Int32,System.Predicate{%60%600})> method overload is used to search the array beginning at position 4 and continuing backward to the beginning of the array. It finds the element at position 1. Finally, the <xref:System.Array.FindLastIndex%60%601(%60%600[],System.Int32,System.Int32,System.Predicate{%60%600})> method overload is used to search the range of three elements beginning at position 4 and working backward (that is, elements 4, 3, and 2). It returns -1 because there are no dinosaur names in that range that end with "saurus".
+
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/Array_FindLastIndex/cpp/source.cpp" id="Snippet1":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Array_FindLastIndex/cs/source.cs" interactive="try-dotnet" id="Snippet1":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/Array_FindLastIndex/vb/source.vb" id="Snippet1":::
+
+ ]]></format>
+        </remarks>
```

Comment:
The sample is not showing. Remarks tag should be inside the Docs tag.
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Examples
 The following code example demonstrates all three overloads of the <xref:System.Array.FindLastIndex%2A> generic method. An array of strings is created, containing 8 dinosaur names, two of which (at positions 1 and 5) end with "saurus". The code example also defines a search predicate method named `EndsWithSaurus`, which accepts a string parameter and returns a Boolean value indicating whether the input string ends in "saurus".

 The <xref:System.Array.FindLastIndex%60%601(%60%600[],System.Predicate{%60%600})> method overload traverses the array backward from the end, passing each element in turn to the `EndsWithSaurus` method. The search stops when the `EndsWithSaurus` method returns `true` for the element at position 5.

> [!NOTE]
>  In C# and Visual Basic, it is not necessary to create the `Predicate<string>` delegate (`Predicate(Of String)` in Visual Basic) explicitly. These languages infer the correct delegate from context and create it automatically.

 The <xref:System.Array.FindLastIndex%60%601(%60%600[],System.Int32,System.Predicate{%60%600})> method overload is used to search the array beginning at position 4 and continuing backward to the beginning of the array. It finds the element at position 1. Finally, the <xref:System.Array.FindLastIndex%60%601(%60%600[],System.Int32,System.Int32,System.Predicate{%60%600})> method overload is used to search the range of three elements beginning at position 4 and working backward (that is, elements 4, 3, and 2). It returns -1 because there are no dinosaur names in that range that end with "saurus".

 :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/Array_FindLastIndex/cpp/source.cpp" id="Snippet1":::
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Array_FindLastIndex/cs/source.cs" interactive="try-dotnet" id="Snippet1":::
 :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/Array_FindLastIndex/vb/source.vb" id="Snippet1":::

 ]]></format>
        </remarks>
      </Docs>
```